scripts/install/install-linux.sh

```diff
@@ -26,5 +26,5 @@ $PORTER_HOME/porter mixin install gcloud --version $PKG_PERMALINK
 $PORTER_HOME/porter plugin install azure --version $PKG_PERMALINK
 
 echo "Installation complete."
-echo "Add porter to your path by running:"
+echo "Add porter to your path by adding the following line to your ~/.bash_profile:"
```

Comment:
I just double checked and on Debian, it comes with `~/.profile` by default instead of `~/.bash_profile`. Let's have them edit that instead.